scripts/scil_score_tractogram.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+from dipy.io.stateful_tractogram import Space, Origin, StatefulTractogram
+from dipy.io.streamline import save_tractogram
+from dipy.tracking.utils import length
+
+import json
+import itertools
+import nibabel as nib
+from nibabel.streamlines import ArraySequence
+import numpy as np
+import os
+import pprint
+from sklearn.cluster import KMeans
+
+from scilpy.image.operations import intersection, difference
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_verbose_arg,
```

Comment:
I would actually use the verbose flag and print logs as files get loaded and the processing goes along. For large dataset, the script takes a while and with no indication on how it's going, it can be a bit confusing.

Reply:
@AntoineTheb Where/which scilpy script can I find an example of this? thanks

Reply:
`scil_recognize_single_bundle` has some form of logging in the VotingScheme class. `scil_decompose_connectivity.py` ouputs messages as the processing goess along.